File: logica-programacao/python/029.py
# Receba um número. Retorne True se ele: for par e maior que 50. Caso contrário, False.

numero = input("Número: ")
float_numero = float(numero)

verificacao = float_numero % 2 == 0 and float_numero > 50

frase_verificacao = f'É par e maior que 50? {verificacao}'

print(frase_verificacao)

File: logica-programacao/javascript/029.js
/* Receba um número. Retorne True se ele: for par e maior que 50. Caso contrário, False. */

let numero = prompt("Número: ");
let floatNumero = parseFloat(numero);

let verificacao = floatNumero % 2 === 0 && floatNumero > 50;

let fraseVerificacao = `É par e maior que 50? ${verificacao}`;

console.log(fraseVerificacao);